task5/script.js

```diff
@@ -0,0 +1,41 @@
+const df = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [
+  {
+    to: "bookmark.html",
+    img: "1.png",
+    alt: "画像1",
+    text: "ブックマーク"
+  },
+  {
+    to: "message.html",
+    img: "2.png",
+    alt: "画像2",
+    text: "メッセージ"
+  }
+];
+
+function arrayPromise() {
+  return new Promise((resolve, reject) => {
+    resolve();
```

Comment:
The problem this time is that we have to "receive the array as a resolved value using a Promise object". What is the resolved value? Normally, the data that was obtained somewhere is used for processing, but this time it is the value that is considered to have solved the processing by omitting it. Currently, nothing is received in the Promise object.